packages/application-shell-connectors/index.js

```diff
@@ -0,0 +1 @@
+export * from './src';
```

Comment:
Nitpick: should we maybe add a comment to those files explaining the reason for it?

Reply:
good idea 👍